crates/proof-of-sql/src/proof_primitive/dory/blitzar_metadata_table.rs

```diff
@@ -153,6 +155,10 @@ pub fn create_blitzar_metadata_tables(
         (0, 0)
     };
 
+    // We will ignore the rows that are zero from the offsets.
+    let offset_row = row_and_column_from_index(offset).0;
```

Comment:
I'm suspicious here.

I think it should be this:
```suggestion
    let offset_row = row_and_column_from_index(offset+1).0-1;
```
Can you add some tests that test with some non-trivial offsets? 0, 1, and 2 are not enough.

I'd like to see tests with non-empty columns for the following offsets at least:
`0,1,2,3,4,5,10,16,20,50,64`
Some fill up rows completely, some do not.

Reply:
Is it possible to add an accessor method instead of using tuple indexing? Its difficult to read and maintain in general.

Reply:
> I'm suspicious here.
> 
> I think it should be this:
> 
> Can you add some tests that test with some non-trivial offsets? 0, 1, and 2 are not enough.
> 
> I'd like to see tests with non-empty columns for the following offsets at least: `0,1,2,3,4,5,10,16,20,50,64` Some fill up rows completely, some do not.

This is super hard to read

Reply:
> (offset+1).0-1;

This is really difficult to read. If its test code its fine but I still am not a fan of this syntax

Reply:
@JayWhite2357 the `max_height` from the `matrix_size` function takes care of offset you are thinking about. It adds the `+1`, so offset doesn't need to worry.

I added tests for the requested offsets.

Reply:
> Is it possible to add an accessor method instead of using tuple indexing? Its difficult to read and maintain in general.

Removed tuple indexing. Thanks!

Reply:
Ok, I'm good with this. The tests proved me wrong, but it still took a while for me to understand why this is correct. `offset` is a length, not an index.